samples/queuesGettingStarted.ts

```diff
@@ -0,0 +1,71 @@
+import { OnMessage, OnError, MessagingError, delay, ServiceBusMessage, ReceiveMode, generateUuid, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let nsSend: Namespace;
+let nsRcv: Namespace;
+async function main(): Promise<void> {
+  await sendMessage();
+  await receiveMessage();
+}
+
+async function sendMessage(): Promise<void> {
+  nsSend = Namespace.createFromConnectionString(str);
+  const sendClient = nsSend.createQueueClient(path);
+  var data = [
+    { name: "Einstein", firstName: "Albert" },
+    { name: "Heisenberg", firstName: "Werner" },
+    { name: "Curie", firstName: "Marie" },
+    { name: "Hawking", firstName: "Steven" },
+    { name: "Newton", firstName: "Isaac" },
+    { name: "Bohr", firstName: "Niels" },
+    { name: "Faraday", firstName: "Michael" },
+    { name: "Galilei", firstName: "Galileo" },
+    { name: "Kepler", firstName: "Johannes" },
+    { name: "Kopernikus", firstName: "Nikolaus" }
+  ];
+
+  for (let i = 0; i < data.length; i++) {
+    var message = {
+      body: JSON.stringify(data[i]),
+      contentType: "application/json",
+      label: "Scientist",
+      messageId: generateUuid(),
+    }
+    await sendClient.send(message);
+    console.log("Sent message number:", i + 1);
+  }
+  console.log("\n>>>>>> Total Sent messages: %d\n", data.length);
+}
+
+async function receiveMessage(): Promise<void> {
+  nsRcv = Namespace.createFromConnectionString(str);
+  const receiveClient = nsRcv.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+  const onMessage: OnMessage = async (brokeredMessage: ServiceBusMessage) => {
+    if (brokeredMessage.label != null &&
```

Comment:
2 questions:
1. should we compare to null here, or undefined? 
2. should this use `!==`  instead of `!=`?

/cc @ramya-rao-a

Reply:
`null != undefined` is false but `null !== undefined` is true. So if `brokeredMessage.label != null` will return false for both when the label is null or undefined.

In this case, we dont need the null check in the first place as the following check on the label text will cover these cases. So both the null checks on the label and the contentType can be removed

Reply:
Ok!